File: android/app/src/main/kotlin/de/bennik2000/dhbwstudentapp/model/ScheduleEntry.kt
package de.bennik2000.dhbwstudentapp.model

import org.threeten.bp.LocalDateTime

class ScheduleEntry(
        val id: Int,
        val title: String,
        val details: String,
        val professor: String,
        val room: String,
        val type: Int,
        val start: LocalDateTime,
        val end: LocalDateTime)

File: ios/dailyschedulewidget/ScheduleEntry.swift
//
//  scheduleentry.swift
//  Runner
//
//  Created by xamarin build on 04.12.20.
//  Copyright © 2020 The Chromium Authors. All rights reserved.
//

import Foundation

public struct ScheduleEntry : Identifiable {
    public let id: Int
    public let start: Date
    public let end: Date
    public let details: String
    public let professor: String
    public let room: String
    public let title: String
    public let type: Int
}
